openmetadata-ui/src/main/resources/ui/src/components/common/Table/Table.tsx

```diff
@@ -311,11 +311,6 @@ const Table = <T extends Record<string, unknown>>(
           ref={ref}
           tableLayout="fixed"
           {...resizingTableProps}
-          scroll={{
```

Comment:
I think we should allow to set scroll from parent no?

Reply:
so, from the rest we are apply the scroll from the parent component.
<img width="493" alt="image" src="https://github.com/user-attachments/assets/b44c3cf3-1bce-4b25-be50-33ce62a096aa" />